### PR TITLE
Fixed typo error in pattern library

### PR DESCRIPTION
**Pull request by vgelani**
## This PR is a:
- [ ] Bug fix or improvement
 

## Summary
 
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.0/pattern-library/filters/data-table-filters/filtering.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Users may remove a filter setting by clicking/tapping the remove control next to
 <img src="img/filter_remove.png">
 
 <h2>Future Capabilities</h2>
-Additional capabilites have been designed to further increase the functionality of filtering, however these features have not, as of yet, been developed.  These include **"Quick Filters"**.
+Additional capabilities have been designed to further increase the functionality of filtering, however these features have not, as of yet, been developed.  These include **"Quick Filters"**.
```
Comment (shrielenee) [inline guides/v2.0/pattern-library/filters/data-table-filters/filtering.md]: Revert this - already addressed

File: guides/v2.0/pattern-library/getting-user-input/date_time_selector/date_time_selector.md [hunk 2]
```diff
@@ -249,7 +249,7 @@ Do not implement keyboard navigation schemes that would place more than one cale
 
 *	The container for the day of week headers and numeric days of the week should have a role of grid
 
-*	The grid should have an aria-labelledby property with a value equivalent to the id of the label for the grid
+*	The grid should have an aria-labelled by property with a value equivalent to the id of the label for the grid
```
Comment (shrielenee) [inline guides/v2.0/pattern-library/getting-user-input/date_time_selector/date_time_selector.md]: Please also apply to v2.1 page.

**Comment by shrielenee** [comment]
Hi @vgelani - Thanks for this typo revision. It looks like the change you made in the `filter.md` file has already been addressed. Can you revert this change for the PR?

The change you included for the `date_time_selector.md` file also needs to be made in the v2.1 version of the pages, which will then propagate to v2.2 and v2.3 as they are symlinked to 2.1. Can you add this change to the v2.1 doc?

File: guides/v2.0/pattern-library/filters/data-table-filters/filtering.md (hunk 1 above)
Comment (vgelani) [reply guides/v2.0/pattern-library/filters/data-table-filters/filtering.md]: Reverted.

File: guides/v2.0/pattern-library/getting-user-input/date_time_selector/date_time_selector.md (hunk 2 above)
Comment (vgelani) [reply guides/v2.0/pattern-library/getting-user-input/date_time_selector/date_time_selector.md]: Applied.

**Comment by vgelani** [comment]
@shrielenee I have reverted back changes and also performed changes in v2.1 `date_time_selector.md` file.

**Comment by shrielenee** [comment]
Thanks, as always, @vgelani!

**Pull request merged**